integration_tests/test_sightings.py

```diff
@@ -173,7 +173,7 @@ def test_sightings(session, login, codex_url, test_root, admin_name):
         'updatedHouston': response.json()['updatedHouston'],
         'version': response.json()['version'],  # 1636456524261
         'creator': {
-            'full_name': 'Test admin',
+            'full_name': my_name,
```

Comment:
possible conflict with my change in #460, probably best to cherry pick my commit for this.

Reply:
Or I take that back, apparently I apparently didn't fix this instance, just the one in the AGS test.

Reply:
That was sort of exploratory to diagnose a different problem. I changed both of them to remove the dependency on the startup test, then I fixed the issue with the startup test so if it'a any problem, this can be undone